docs/_config.yml

```diff
@@ -13,3 +13,4 @@ color_scheme: firefly
 plugins:
   - jekyll-multiple-languages-plugin
 languages: ["en"]
+google_tag_manager: GTM-N58535C
```

Comment:
We might need to swap this out for a different key. I'm checking on this to see if Hyperledger has one we can use.

Reply:
Awesome, thanks!